recipes-bsp/atf/atf_git.bb

```diff
@@ -27,6 +27,9 @@ PLATFORM_ls1088ardb-pb = "ls1088ardb"
 PLATFORM_ADDITIONAL_TARGET ??= ""
 PLATFORM_ADDITIONAL_TARGET_ls1012afrwy = "ls1012afrwy_512mb"
 
+RCW_FOLDER ?= "${MACHINE}"
+RCW_FOLDER_ls1088ardb-pb = "${MACHINE}"
```

Comment:
The override is the very same, so  please avoid it.

Reply:
it is ls1088ardb, not ${MAHICNE}. A new commit is pushed to fix it.

Reply:
Squash it.

Reply:
squashed.